src/e2e/e2e.test.js

```diff
@@ -21,10 +21,18 @@ describe("Beat", () => {
       args: ["--start-fullscreen"],
     });
     page = await browser.newPage();
-    await page.goto(`${CLIENT_URL}/login`);
   });
 
-  afterAll(() => {
+  afterAll(async () => {
+    await page.evaluate((email) => {
+      fetch(`http://localhost:8000/api/users/delete`, {
```

Comment:
```suggestion
      fetch(`env/api/users/delete`, {
```
env처리를 해주시면 좋을꺼 같습니다!